packages/app-extensions/src/appFactory/ThemeWrapper.js

```diff
@@ -19,6 +19,10 @@ class ThemeWrapper extends React.PureComponent {
   }
 }
 
+ThemeWrapper.defaultProps = {
```

Comment:
good. but you should adjust line 11:

`    const mergedTheme = _merge({}, theme, appTheme)`